Initial version to install

loicbenoit opened the pull request.

loicbenoit committed:
Initial version

loicbenoit committed:
Add files via upload

loicbenoit committed:
Add files via upload

loicbenoit committed:
Add files via upload

loicbenoit committed:
Add files via upload

loicbenoit committed:
Add files via upload

loicbenoit committed:
Add files via upload

Commit:
Add files and correct some mistakes

Commit:
added missing files

loicbenoit committed:
Created readme.md

psionikangel commented:
@euzhanregis Est-ce que tu es en mesure de réviser et merger cette PR en l'absence de @fredhamel?

loicbenoit committed:
Bug fix

loicbenoit committed:
Fix composer install/update

Add custom repository for more direct update path. Remove unwanted glpi-dev packages.
